src/modules/trade/hooks/useNativeCurrency.ts

```diff
@@ -0,0 +1,16 @@
+import { useMemo } from 'react'
+
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { NativeCurrency } from '@uniswap/sdk-core'
+
+import { GpEther as ETHER } from 'legacy/constants/tokens'
+import { supportedChainId } from 'legacy/utils/supportedChainId'
+
+import { useWalletInfo } from 'modules/wallet'
+
+export function useNativeCurrency(): NativeCurrency {
```

Comment:
We already have smth similar:
```
import useNativeCurrency from 'lib/hooks/useNativeCurrency'
```

Isn't it the same?

Reply:
Yeah, I marked that one as deprecated because it uses a slightly different logic and is part of the libs module.
But maybe we can just use that instead. I'll try it out and remove if so.